GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -1594,8 +1595,8 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                             uint radius = 1000; // Assumed radius
                             (long start, long end) lifespan = effect.ComputeLifespan(log, 2500); // Assumed duration
                             var positionConnector = new PositionConnector(effect.Position);
-                            var shockwave = (CircleDecoration)new CircleDecoration(radius, lifespan, Colors.LightGrey, 0.5, positionConnector).UsingFilled(false).UsingGrowingEnd(lifespan.end);
-                            replay.Decorations.Add(shockwave);
+                            GeographicalConnector connector = new PositionConnector(effect.Position);
+                            replay.AddShockwave(connector, lifespan, Colors.LightGrey, 0.6, radius);
```

Comment:
0.5 opacity

Reply:
I've changed the opacity of some because I've moved from the string colors to the class ones, so I've tried to match them to what they were that way

Reply:
Ok, the PR did not clarify that some colorimetric changes were being made, in that case disregard my comments regarding colors and opacities.